docs/application/web/api/6.5/device_api/mobile/tizen/application.html

```diff
@@ -3204,6 +3204,7 @@ <h4>Methods</h4>
 <div class="description">
           <p>
 System events do not require an application identifier to be specified. If one is specified, the event will be interpreted as an user event.
+List of supported system event is available in <a href="https://docs.tizen.org/application/native/guides/app-management/event/#platform-event-types">Native API guides</a>.
```

Comment:
FYI. In case that the page in docs.tizen.org are linked, it is recommend to use relative path.